Error calling example RE pipeline: AssertionError: No argument markers available, was `prepare` already called?

Version 0.13.0. Sample code is essentially identical to https://huggingface.co/spaces/pie/Joint-NER-and-Relation-Extraction/blob/main/app.py:

```python
def main(text):

    ner_model_name_or_path = "pie/example-ner-spanclf-conll03"
    re_model_name_or_path = "pie/example-re-textclf-tacred"

    ner_pipeline = AutoPipeline.from_pretrained(ner_model_name_or_path, device=-1)
    re_pipeline = AutoPipeline.from_pretrained(re_model_name_or_path, device=-1)

    document = ExampleDocument(text)

    ner_pipeline(document)

    while len(document.entities.predictions) > 0:
        entity = document.entities.predictions.pop(0)
        print(entity)
        document.entities.append(entity)

    re_pipeline(document)
    for relation in document.relations.predictions:
        print(relation.head, relation.tail, relation.label)
```

Running this on a simple text document gives me an error in `re_pipeline()`:

```
Traceback (most recent call last):
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/test.py", line 53, in <module>
    main(txt)
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/test.py", line 37, in main
    re_pipeline(document)
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/pytorch-ie-venv/lib/python3.9/site-packages/pytorch_ie/pipeline.py", line 342, in __call__
    model_inputs = self.preprocess(documents, **preprocess_params)
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/pytorch-ie-venv/lib/python3.9/site-packages/pytorch_ie/pipeline.py", line 216, in preprocess
    encodings = self.taskmodule.encode(
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/pytorch-ie-venv/lib/python3.9/site-packages/pytorch_ie/core/taskmodule.py", line 339, in encode
    cur_task_encodings, cur_documents_in_order = self.batch_encode(
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/pytorch-ie-venv/lib/python3.9/site-packages/pytorch_ie/core/taskmodule.py", line 247, in batch_encode
    task_encodings, documents_in_order = self.encode_inputs(
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/pytorch-ie-venv/lib/python3.9/site-packages/pytorch_ie/core/taskmodule.py", line 378, in encode_inputs
    possible_task_encodings = self.encode_input(document, is_training)
  File "/Users/sam/Projects/KBP-FtMeade-22/extractors/pytorch-ie/pytorch-ie-venv/lib/python3.9/site-packages/pytorch_ie/taskmodules/transformer_re_text_classification.py", line 259, in encode_input
    assert (
AssertionError: No argument markers available, was `prepare` already called?
```

My setup is Python 3.9 on MacOS. Clean venv in which I ran

`pip install pytorch-ie`

Encountered an error when importing `pytorch-ie` in my code: 

`ImportError: cannot import name 'CommitOperationAdd' from 'huggingface_hub' (/[deleted]/pytorch-ie-venv/lib/python3.9/site-packages/huggingface_hub/__init__.py)`

huggingface_hub version was 0.5.1, ran `pip install --upgrade huggingface-hub`, which warned me that `pytorch-ie` expected nothing higher than 0.5.1, but got me past that error, only to hit the assertion error above.

Thanks in advance.

--- a/src/pytorch_ie/auto.py
+++ b/src/pytorch_ie/auto.py
@@ -26,7 +26,9 @@ class AutoTaskModule(PyTorchIETaskmoduleModelHubMixin):
     ) -> TaskModule:
         class_name = module_kwargs.pop(cls.config_type_key)
         clazz = TaskModule.by_name(class_name)  # type: ignore
-        return clazz(**module_kwargs)
+        taskmodule: TaskModule = clazz(**module_kwargs)
+        taskmodule._post_prepare()
+        return taskmodule
 
 
 class AutoModel(PyTorchIEModelHubMixin):

--- a/tests/pipeline/test_ner_span_classification.py
+++ b/tests/pipeline/test_ner_span_classification.py
@@ -2,6 +2,7 @@ from dataclasses import dataclass
 
 import pytest
 
+from pytorch_ie import AutoPipeline
 from pytorch_ie.annotations import LabeledSpan
 from pytorch_ie.core import AnnotationList, annotation_field
 from pytorch_ie.documents import TextDocument
@@ -17,16 +18,23 @@ class ExampleDocument(TextDocument):
 
 @pytest.mark.slow
 @pytest.mark.parametrize("fast_dev_run", [False, True])
-def test_ner_span_classification(fast_dev_run):
+@pytest.mark.parametrize("use_auto", [False, True])
+def test_ner_span_classification(fast_dev_run, use_auto):
     model_name_or_path = "pie/example-ner-spanclf-conll03"
-    ner_taskmodule = TransformerSpanClassificationTaskModule.from_pretrained(model_name_or_path)
-    assert ner_taskmodule.is_from_pretrained
-    ner_model = TransformerSpanClassificationModel.from_pretrained(model_name_or_path)
-    assert ner_model.is_from_pretrained
-
-    ner_pipeline = Pipeline(
-        model=ner_model, taskmodule=ner_taskmodule, device=-1, fast_dev_run=fast_dev_run
-    )
+    if use_auto:
+        ner_pipeline = AutoPipeline.from_pretrained(
+            model_name_or_path, device=-1, fast_dev_run=fast_dev_run
+        )
+    else:
+        ner_taskmodule = TransformerSpanClassificationTaskModule.from_pretrained(
+            model_name_or_path
+        )
+        ner_model = TransformerSpanClassificationModel.from_pretrained(model_name_or_path)
+        ner_pipeline = Pipeline(
+            model=ner_model, taskmodule=ner_taskmodule, device=-1, fast_dev_run=fast_dev_run
+        )
+    assert ner_pipeline.taskmodule.is_from_pretrained
+    assert ner_pipeline.model.is_from_pretrained
 
     text = "“Making a super tasty alt-chicken wing is only half of it,” said Po Bronson, general partner at SOSV and managing director of IndieBio."
     documents = [ExampleDocument(text), ExampleDocument(text), ExampleDocument(text)]

--- a/tests/pipeline/test_re_text_classification.py
+++ b/tests/pipeline/test_re_text_classification.py
@@ -3,6 +3,7 @@ from typing import Sequence
 
 import pytest
 
+from pytorch_ie import AutoPipeline
 from pytorch_ie.annotations import BinaryRelation, LabeledSpan
 from pytorch_ie.core import AnnotationList, annotation_field
 from pytorch_ie.documents import TextDocument
@@ -18,14 +19,19 @@ class ExampleDocument(TextDocument):
 
 
 @pytest.mark.slow
-def test_re_text_classification():
+@pytest.mark.parametrize("use_auto", [False, True])
+def test_re_text_classification(use_auto):
     model_name_or_path = "pie/example-re-textclf-tacred"
-    re_taskmodule = TransformerRETextClassificationTaskModule.from_pretrained(model_name_or_path)
-    assert re_taskmodule.is_from_pretrained
-    re_model = TransformerTextClassificationModel.from_pretrained(model_name_or_path)
-    assert re_model.is_from_pretrained
-
-    pipeline = Pipeline(model=re_model, taskmodule=re_taskmodule, device=-1)
+    if use_auto:
+        pipeline = AutoPipeline.from_pretrained(model_name_or_path)
+    else:
+        re_taskmodule = TransformerRETextClassificationTaskModule.from_pretrained(
+            model_name_or_path
+        )
+        re_model = TransformerTextClassificationModel.from_pretrained(model_name_or_path)
+        pipeline = Pipeline(model=re_model, taskmodule=re_taskmodule, device=-1)
+    assert pipeline.taskmodule.is_from_pretrained
+    assert pipeline.model.is_from_pretrained
 
     document = ExampleDocument(
         "“Making a super tasty alt-chicken wing is only half of it,” said Po Bronson, general partner at SOSV and managing director of IndieBio."

